src/commonMain/kotlin/exchange.dydx.abacus/state/manager/StateManagerAdaptor.kt

```diff
@@ -1802,15 +1802,11 @@ open class StateManagerAdaptor(
         val reduceOnly = true
         val postOnly = false
 
-        val timeInForce = when (triggerOrder.type) {
-            OrderType.stopMarket, OrderType.takeProfitMarket -> "IOC"
-            OrderType.stopLimit, OrderType.takeProfitLimit -> "GTT"
-            else -> throw Exception("invalid triggerOrderType")
-        }
+        val timeInForce = null;
 
         val execution = when (triggerOrder.type) {
             OrderType.stopMarket, OrderType.takeProfitMarket -> "IOC"
-            OrderType.stopLimit, OrderType.takeProfitLimit -> "Default"
+            OrderType.stopLimit, OrderType.takeProfitLimit -> "DEFAULT"
```

Comment:
Should these instead be an enum also?

Reply:
I've been curious about this in the abacus codebase - we seem to use strings everywhere (instead of enums), not sure if there's a historical reason for this (was just pattern matching)

Reply:
Be the change :) From a brief look probably fine to keep as is in this PR, but would be good to perhaps explore updating some or all of the `String` fields in `HumanReadablePlaceOrderPayload` (TIF, execution, type, side).

Reply:
will explore in a follow up!